weblate/vcs/git.py

```diff
@@ -563,18 +578,20 @@ def push_to_fork(self, local_branch, fork_branch):
         """Push given local branch to branch in forked repository."""
         self.execute(
             [
+                "git",
```

Comment:
I think it's better to completely remove hub usage from the GitHub class instead of rewriting it here. Right now it might be probably broken for GitLab which still relies on `lab`, but it would not be executed (not sure if that is needed though).